calicoctl/commands/node/run.go

```diff
@@ -303,15 +303,7 @@ Description:
 			vol{hostPath: "/var/run/docker.sock", containerPath: "/var/run/docker.sock"})
 	}
 
-	if etcdcfg.EtcdEndpoints == "" {
```

Comment:
For the CLI output I think we should convert the authority and scheme to an endpoint.  This means the output from the CLI will *never* show the ETCD_AUTHORITY environment.  If a user has already constructed their `docker run` incantation using ETCD_AUTHORITY then that will still work - but we shouldn't advertise that when creating a new command using `calicoctl node run`